src/GraphQL.Tests/ObjectExtensionsTest.cs

```diff
@@ -0,0 +1,70 @@
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+using GraphQL.Types;
+using Shouldly;
+using Xunit;
+
+namespace GraphQL.Tests
+{
+    public class ObjectExtensionsTests
+    {
+        [Theory]
+        [ClassData(typeof(CultureList))]
+        public void convert_double(CultureInfo cultureInfo)
+        {
+            CultureInfo.DefaultThreadCurrentCulture = cultureInfo;
+            CultureInfo.DefaultThreadCurrentUICulture = cultureInfo;
```

Comment:
I think we should use something like this [`UseCulture`](https://github.com/xunit/samples.xunit/blob/master/UseCulture/UseCultureAttribute.cs) attribute since this sets the culture but then does not revert it back to its original value.  That could then create instability between tests.